layers/+source-control/git/packages.el

```diff
@@ -25,6 +25,7 @@
         (helm-git-grep :requires helm)
         (helm-gitignore :requires helm)
         magit
+        (magit-delta :if git-enable-magit-delta-plugin)
```

Comment:
This will not prevent installing this package, the keyword you need is `:toggle` like in `(dash-at-point :toggle (spacemacs/system-is-mac))`. Just tested this and without me setting the new variable the package has been installed.

Reply:
Hi @smile13241324 , I hope to have addressed your comments. Would you mind reviewing and merging/cherry-picking when you have a chance?

Thanks,